Assets/_BForBoss/_Core/Scripts/HUD/EnergySystemViewBehaviour.cs

```diff
@@ -18,7 +19,12 @@ public void Initialize(IEnergyDataSubject energyDataSubject)
 
         private void EnergyDataSubjectOnOnStateChanged(EnergyData data)
         {
-            _fillImage.fillAmount = data.Value / Math.Max(data.MaxEnergyValue, 1f);
+            _targetFillAmount = data.Value / Math.Max(data.MaxEnergyValue, 1f);
+        }
+
+        private void Update()
+        {
+            _fillImage.fillAmount = Mathf.Lerp(_fillImage.fillAmount, _targetFillAmount, Time.unscaledDeltaTime * 10f);
```

Comment:
could you make 10.0f a const variable

Reply:
oh yeah. sure

Reply:
[370ec0c](https://github.com/PerigonGames/BForBoss/pull/332/commits/370ec0c1bdc39d118324912d00dfb6e2bc2832c6)

Update